k8s/src/main/scala/io/buoyant/k8s/istio/RouteManager.scala

```diff
@@ -1,28 +1,30 @@
-package io.buoyant.k8s
+package io.buoyant.k8s.istio
 
 import com.twitter.conversions.time._
 import com.twitter.finagle.Http
 import com.twitter.finagle.param.Label
 import com.twitter.finagle.tracing.NullTracer
 import com.twitter.util._
-import io.buoyant.k8s.IstioPilotClient.RouteRuleConfig
+import io.buoyant.k8s.SetHostFilter
+import io.buoyant.k8s.istio.ApiserverClient.RouteRuleConfig
 import io.buoyant.namer.RichActivity
 import istio.proxy.v1.config.RouteRule
 
-class RouteManager(api: IstioPilotClient, pollInterval: Duration) {
+class RouteManager(api: ApiserverClient) extends Closable {
```

Comment:
rename this to `RouteCache` for consistency (or renamed `ClusterCache` to `ClusterManager`)